docs/Contributing.asciidoc

```diff
@@ -481,6 +481,16 @@ createdb -O ownername -T originaldb newdb
 To initialize or update the database manually before restarting the web UI you can run
 either `./script/initdb --init_database` or `./script/upgradedb --upgrade_database`.
 
+Migrations that affect possibly big tables should be tested against a production
```

Comment:
Not against production database but rather against production data locally, right?